packages/manager/modules/pci/src/projects/project/private-registry/private-registry.service.js

```diff
@@ -0,0 +1,82 @@
+import head from 'lodash/head';
+import assign from 'lodash/assign';
+
+export default class pciPrivateRegistryService {
+  /* @ngInject */
+  constructor(
+    OvhApiCloudProject,
+    OvhApiMe,
+  ) {
+    this.OvhApiPrivateRegistry = OvhApiCloudProject.ContainerRegistry().v6();
+    this.OvhApiPrivateRegistryUser = OvhApiCloudProject.ContainerRegistry().Users().v6();
+    this.User = OvhApiMe;
+  }
+
+  /**
+   * create a new registry
+   * @accepts registry name
+   * @returns new registry
+   * @memberof pciPrivateRegistryService
+   */
+  createRegistry(projectId, registry) {
```

Comment:
I think it's not necessary to specify registry each time.

`pciPrivateRegistry.create()` is explicit :)